lxd/daemon.go

```diff
@@ -573,13 +573,13 @@ func (d *Daemon) Init() error {
 
 	/* Print welcome message */
 	if d.MockMode {
-		shared.LogInfo("LXD is starting in mock mode",
+		shared.LogDebug("LXD is starting in mock mode",
```

Comment:
Those 3 I'd keep at LogInfo as they are very useful messages to see in user submitted logs.